package.json

```diff
@@ -26,7 +26,7 @@
     "optimize_news_images": "bash ./optimize_images.sh content/news"
   },
   "dependencies": {
-    "eclipsefdn-hugo-solstice-theme": "0.0.181",
+    "eclipsefdn-hugo-solstice-theme": "0.0.186",
```

Comment:
We should also update the Hugo version in the netlify file if we upgrade this package. See https://gitlab.eclipse.org/eclipsefdn/infrazilla/-/issues/1011 for more info.

We also need to update the readme.md when we make this change:
https://gitlab.eclipse.org/eclipsefdn/it/websites/eclipse.org/-/commit/f85703d8ff207a74cdc4ca0a3bd2e02549d52951

Finally, we had some issues with the spec project:

TLDR; yarn is not installed for that project. Can you also switch form yarn.lock to packages-lock.json with this change?